### PR TITLE
[TSI] Rename Iot to IoT for namespace of TSI types

### PR DESCRIPTION
**Pull request by bikamani**
Renaming Iot to IoT

### CODE REVIEW

File: sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/api/Azure.IoT.TimeSeriesInsights.netstandard2.0.cs [hunk 1]
```diff
@@ -363,10 +363,16 @@ public partial class TimeSeriesInsightsClient
...
         public virtual Azure.Response<Azure.IoT.TimeSeriesInsights.TimeSeriesOperationError[]> CreateOrReplaceTimeSeriesInstances(System.Collections.Generic.IEnumerable<Azure.IoT.TimeSeriesInsights.TimeSeriesInstance> timeSeriesInstances, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Response<Azure.IoT.TimeSeriesInsights.TimeSeriesOperationError[]>> CreateOrReplaceTimeSeriesInstancesAsync(System.Collections.Generic.IEnumerable<Azure.IoT.TimeSeriesInsights.TimeSeriesInstance> timeSeriesInstances, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<Azure.IoT.TimeSeriesInsights.TimeSeriesOperationError[]> CreateOrReplaceTimeSeriesTypes(System.Collections.Generic.IEnumerable<Azure.IoT.TimeSeriesInsights.TimeSeriesType> timeSeriesTypes, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```
Comment (bikamani) [inline sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/api/Azure.IoT.TimeSeriesInsights.netstandard2.0.cs:366]: This should fix a bad merge which had overridden this change.

File: sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/TimeSeriesInsightsClient.cs [hunk 2]
```diff
@@ -1184,6 +1184,157 @@ public virtual Response<SearchSuggestion[]> GetSearchSuggestions(string searchSt
             }
         }
 
+        private AsyncPageable<QueryResultPage> QueryEventsInternalAsync(
```
Comment (bikamani) [inline sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/TimeSeriesInsightsClient.cs:1187]: rearranging code for readability

Comment (barustum) [reply sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/TimeSeriesInsightsClient.cs:1187]: The usual convention for C# is to place private methods towards the end of the file, regardless of where they are being called. So private methods usually go towards the bottom of the file.
It seems like StyleCop does it this was as well:
https://documentation.help/StyleCop/SA1202.html

Curious to know how the team does things on other projects.

Comment (bikamani) [reply sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/TimeSeriesInsightsClient.cs:1187]: Interesting and in a way it makes sense. I don't have a strong opinion and can change it.

**Review by barustum — APPROVED** [review]

**Pull request merged**